lib/ttnt/testtask.rb

```diff
@@ -12,14 +13,21 @@ class TestTask
     # An instance of `Rake::TestTask` passed when TTNT::TestTask is initialized
     attr_accessor :rake_testtask
 
+    attr_accessor :test_files
+    attr_accessor :code_files
```

Comment:
What do you think about doing:

~~~ruby
attr_reader :test_files
attr_reader :code_files

def code_files=(files)
  @code_files = files.kind_of?(String) ? FileList[files] : files
end

def test_files=(files)
  @test_files = files.kind_of?(String) ? FileList[files] : files
end
~~~